community/groff/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+
+patch -p0 < musl.patch
+
+./configure \
+	--prefix=/usr \
+	--without-x \
+	--disable-rpath \
+	--with-doc=no \
+	--disable-nls
```

Comment:
Change this to `--enable-nls=auto` so that users with `gettext` can have `groff` link to it. :+1: 

If this is the default behavior, this line can be removed.